frida_tools/tracer.py

```diff
@@ -346,19 +346,38 @@ def _create_resolver_script(self):
 }
 
 function includeFunction(pattern, workingSet) {
-    moduleResolver().enumerateMatches('exports:*!' + pattern).forEach(function (m) {
+    var parseResult = parseExportsFunctionPattern(pattern);
+    moduleResolver().enumerateMatches('exports:' + parseResult.module + '!' + parseResult.function).forEach(function (m) {
         workingSet[m.address.toString()] = moduleExportFromMatch(m);
     });
     return workingSet;
 }
 
 function excludeFunction(pattern, workingSet) {
-    moduleResolver().enumerateMatches('exports:*!' + pattern).forEach(function (m) {
+    var parseResult = parseExportsFunctionPattern(pattern);
+    moduleResolver().enumerateMatches('exports:' + parseResult.module + '!' + parseResult.function).forEach(function (m) {
         delete workingSet[m.address.toString()];
     });
     return workingSet;
 }
 
+function parseExportsFunctionPattern(pattern) {
+    var res = pattern.split('!');
+    var m, f;
+    
```

Comment:
There's still trailing whitespace here. I think we should keep a blank line here though (just remove the whitespace), and move `var m, f;` right after it.

Reply:
OK, modified as per your request.  Question: why the blank line between the `res` and `m,f` definitions.  Since both are used in the next if-block, shouldn't they be adjacent to each other with no intervening blank line?